Makefile

```diff
@@ -1,14 +1,14 @@
 COMMIT=$(shell git rev-parse HEAD)
 VERSION=$(shell git describe --tags --exact-match --always)
 DATE=$(shell date +'%FT%TZ%z')
+CONTAINER_NAME ?= vegeta
 
 SERVER_DIR = cmd/server
-GO  = GO111MODULE=on go
```

Comment:
So I get that Travis would handle this correctly, but what about running this on a local system. For `go mod` to work we need to set `GO111MODULE` (not sure if this is the case in 1.12 though) right ?

Reply:
Actually no, as long as `go.mod` is present in the workdir this defaults to `GO111MODULE=on`. More on this can be found [here](https://blog.golang.org/modules2019).